### PR TITLE
MDOT-DLC API Integration Fixes

### PR DESCRIPTION
**Pull request by camerontesterman**
## Original Issue
[link to original issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/7507)

## Description of change
This PR includes a few changes to successfully connect with the stand-in DLC API. These changes include:

- Altering the `supplies.json` to allow some fields to be nullable, such as temporary address as not every veteran will have a temporary address.
- Changing the client connection URL to make calls to `/supplies` instead of `/mdot/supplies`

## Testing
Has been tested and verified using a locally running instance of vets-api

#### Unique to this PR
- [x] Calls made using the MDOT Client return data successfully when connected to the DLC stand-in API

#### Applies to all PRs

- [x] Swagger docs have been updated, if applicable
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [x] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

File: lib/mdot/client.rb [hunk 1]
```diff
@@ -59,7 +59,10 @@ def submit_order(request_body)
     private
 
     def headers
-      { veteranId: @user.ssn }
+      {
+        VA_VETERAN_ID: @user.ssn.split(//).last(4).join,
```
Comment (lihanli) [inline lib/mdot/client.rb]: how about `@user.ssn.last(4)`?

Comment (camerontesterman) [reply lib/mdot/client.rb]: gotta love that rails magic, tested this in irb and it did not work, but sure enough in the rails console it does. changing it.

**Comment by jholton** [comment]
> Provide link to originating GitHub issue, or connected to it via ZenHub
Is the issue linked?  I don't see it.

**Comment by camerontesterman** [comment]
> > Provide link to originating GitHub issue, or connected to it via ZenHub
> 
> Is the issue linked? I don't see it. @camerontesterman

Has been added at the top. Here is a [link](https://github.com/department-of-veterans-affairs/va.gov-team/issues/7507) as well.

**Comment by jholton** [comment]
Thanks!

**Pull request merged**